### PR TITLE
[Docs] Update connections.md

### PR DESCRIPTION
**Pull request by dev-test-01**
**Notes for Reviewers**

This PR fixes typos in Meshery docs

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a7f3bbd89750922346afaecb4b6b42d4cdcadb15 at: https://66e82faf94e4b13e201c40a7--meshery-docs-preview.netlify.app

File: docs/pages/concepts/logical/connections.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ When a connection has been discovered, it will be listed in the MeshSync browser
...
 
-> EExampleg: User manually selects the registered Prometheus connection and transition to the [connected](#state-connected) state (i.e. User administratively processes the connection).
+> Example: User manually selects the registered Prometheus connection and transition to the [connected](#state-connected) state (i.e. User administratively processes the connection).
 
 ### State: Connected
 
-The connection in this state is administratively processed and being actively managed by Meshery. User can interface and invoke set of actions with the connection.</br>
-From this state the transition can happen to either [Maintenance](#state-maintenance) or [Ignore](#state-ignored) state. </br> Auto transition to [Disconnected](#state-disconnected) state will occur if Meshery can no longer communicate with the connection, which can occur due to connectivity issue/AuthN-AuthZ/connection was deleted outside Meshery or any other issue.
+The connection in this state is administratively processed and being actively managed by Meshery. User can interface and invoke set of actions with the connection.
```
Comment (dev-test-01) [inline docs/pages/concepts/logical/connections.md:40]:
```suggestion
The connection in this state is administratively processed and being actively managed by Meshery. User can interface and invoke set of actions with the connection.<br>
```

File: docs/pages/concepts/logical/connections.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ When a connection has been discovered, it will be listed in the MeshSync browser
...
+From this state the transition can happen to either [Maintenance](#state-maintenance) or [Ignore](#state-ignored) state. Auto transition to [Disconnected](#state-disconnected) state will occur if Meshery can no longer communicate with the connection, which can occur due to connectivity issue/AuthN-AuthZ/connection was deleted outside Meshery or any other issue.
```
Comment (dev-test-01) [inline docs/pages/concepts/logical/connections.md:41]:
```suggestion
From this state the transition can happen to either [Maintenance](#state-maintenance) or [Ignore](#state-ignored) state.<br>
Auto transition to [Disconnected](#state-disconnected) state will occur if Meshery can no longer communicate with the connection, which can occur due to connectivity issue/AuthN-AuthZ/connection was deleted outside Meshery or any other issue.
```

**Comment by l5io** [comment]
🚀 Preview for commit 81b161e7a3944765bc5da14ece3fc911ab654ba9 at: https://66e838ca6016af00708687a3--meshery-docs-preview.netlify.app

**Review by vishalvivekm — APPROVED** [review]
Thank you @dev-test-01

**Pull request merged**